pkg/testing/e2e_test.go

```diff
@@ -191,3 +191,126 @@ func (s *E2ESuite) TestKubeExec(c *C) {
 	})
 	c.Assert(err, IsNil)
 }
+
+func (s *E2ESuite) TestKubeTask(c *C) {
+	ctx, can := context.WithTimeout(context.Background(), 30*time.Second)
+	defer can()
+
+	// Create a test Deployment
+	d, err := s.cli.AppsV1().Deployments(s.namespace).Create(testutil.NewTestDeployment(1))
+	c.Assert(err, IsNil)
+	err = kube.WaitOnDeploymentReady(ctx, s.cli, s.namespace, d.GetName())
+	c.Assert(err, IsNil)
+
+	// Create a dummy Profile and secret
+	sec := &v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "test-secret-",
+		},
+		StringData: map[string]string{
+			"id":  "foo",
+			"key": "bar",
+		},
+	}
+	sec, err = s.cli.CoreV1().Secrets(s.namespace).Create(sec)
+	c.Assert(err, IsNil)
+	p := &crv1alpha1.Profile{
```

Comment:
@PrasadG193  can we make use of func `NewTestProfile(namespace string, secretName string)` from  pkg/testutil/testutil.go  here ?